CONTRIBUTING.md

```diff
@@ -12,6 +12,24 @@ Everything is formatted with `dart format` (no flags) and linted with `dart anal
 
 ## Translations
 
+### Weblate
+
+Lemmy devs are kindly hosting lemmur translation strings on their [Weblate instance](https://weblate.yerbamate.ml/projects/lemmur/lemmur/). Feel free to contribute strings there, we regularly sync string changes with Weblate.
+
+We use flutter's native file format for translations: ARB. In most cases you will be able to deduce the syntax based on the source string. Here are 3 important examples:
+
+1. Placeholders
+
+`Hello there {name}!` - placeholders are put in a pair of braces, it will be later replaced with an appropriate value.
+
+2. Plurals
+
+`You have {amount} new {amount, plural, =0{messages} =1{message} =2{messages} few{messages} many{messages} other{message}}` - plurals are checked against their quantifier and provide 6 possible forms to choose from. In english this example does not make much sense, since we could just provide the `=1{message}` and `other{messages}` case. `other` case always has to be specified, it acts as a fallback.
+
+3. Selects
+
+`I will take a {distance_name, select, close{bus} far{train} veryFar{plane}}.` - selects allow for arbitrary matching against some predefined cases. All cases should be the same as in the source string.
```

Comment:
you should mention that the strings follow ICU specification so that it's more clear to the reader, and that they are able to read more about it